content/millennium/r4/medications/immunization.md

```diff
@@ -0,0 +1,118 @@
+* [Administration date](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.occurrence_x_){:target="_blank"}
+* [Indicates if the record is from the source who administered the vaccine](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.primarySource){:target="_blank"}
+* [Source of secondarily reported record](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.reportOrigin){:target="_blank"}
+* [Location](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.location){:target="_blank"}
+* [Vaccine manufacturer](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.manufacturer){:target="_blank"}
+* [Vaccine lot number](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.lotNumber){:target="_blank"}
+* [Vaccine expiration date](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.expirationDate){:target="_blank"}
+* [Vaccine administration site](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.site){:target="_blank"}
+* [Vaccine administration route](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.route){:target="_blank"}
+* [Dosage administered](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.doseQuantity){:target="_blank"}
+* [Performer](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.performer){:target="_blank"}
+    * [Type of performer (ordering or administering)](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.performer.function){:target="_blank"}
+    * [Who performed](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.performer.actor){:target="_blank"}
+* [Note](http://hl7.org/fhir/R4/immunization-definitions.html#Immunization.note){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:immunization, :r4) %>
+
+## Search
+
+Search for Immunizations that meet supplied query parameters:
+
+    GET /Immunization?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`     | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient` | This, or `_id`     | [`reference`] | The patient for the vaccination record. Example: `12345`
+ `date`    | N                  | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
+
+Notes:
+ - The `date` parameter value should be prefixed once by 'ge' representing the earliest date, and once by 'le' representing the latest date. Examples: `date=ge2015-01-01&date=le2016-01-01`, `date=ge2015-01-01`, `date=le2016-01-01`
```

Comment:
need to remove the leading space before the dash character so this is rendered as a bullet

can you switch the dash to an asterisk?

can you fix this in content/millennium/dstu2/medications/immunization.md too?

Reply:
Fixed in both the places.

R4: https://github.com/cerner/fhir.cerner.com/pull/255/commits/7f7bd5fd0fd7d7d18677ca1d5618107ccb707330#diff-3e96067fd99a2fee85880712d23a6719R65

DSTU2: https://github.com/cerner/fhir.cerner.com/pull/255/commits/7f7bd5fd0fd7d7d18677ca1d5618107ccb707330#diff-7154922f6909b151245502b6c0e575f5R59